TripalCultivate/TripalCultivate-Phenotypes
# G4.91 remove old design remnants

**Issue #91 **

## Motivation



This Draft PR will test to see if the validation result feedback engine will render the validation result status array into the validation result window in the frontend.

<img width="966" alt="image" src="https://github.com/user-attachments/assets/0b723347-ce3b-464e-a460-0a15275028b7">


## What does this PR do?
*Please describe each things this PR does. For example, a PR may 1) solve a specific bug, 2) create an auomated test to ensure it doesn't return.*

1. Create a variable that will be set on the first instance of failed validation. Subsequent validation will check this variable to see if status will be set to TODO
2. Minor revision to the method to reference the predefined validation array


## Testing

### Automated Testing
*Please describe each automated test this PR creates and provide a list of the assertions it makes using casual language.*
*Do not just say things like "asserts the array is not empty" but rather say "Ensures that the return value of method X with these parameters is not an empty array".*

ClassName->testSpecificBug: tests this specific bug
- 
- 
- 

### Manual Testing
*Describe in detail how someone should manually test this functionality.*
*Make sure to include whether they need to build a docker from scratch, create any records, etc.*

1. In the example shown above, upload a test trait file where the Type header is set to neither Quantitative nor Qualitative.
